src/components/OrderItem/OrderItem.jsx

```diff
@@ -23,8 +23,8 @@ export class OrderItem extends React.Component {
           <span className="order-item__price"><Price value={price} /></span>
         </p>
         {isHot && <small className="hot">HOT</small>}
-        {bread !== "Standard" && <small>{bread}</small>}
-        <button onClick={this.action}>- Remove</button>
+        {bread !== 'Standard' && <small>{bread}</small>}
+        <button type="button" onClick={this.action}>- Remove</button>
```

Comment:
The `onClick` of the button would then become `onClick={() => removeFromOrder(id)}`